### PR TITLE
[12.0][ADD] hr_expense_advance_clearing_sequence

### PR DESCRIPTION
**Pull request by ps-tubtim**
This PR is waiting for https://github.com/OCA/hr/pull/621, you can always use this for readability.

- [x]  [12.0][MIG] hr_expense_sequence

### CODE REVIEW

**Comment by ps-tubtim** [comment]
cc: @kittiu

**Review by Saran440 — APPROVED** [review]
Code Reviewed. LGTM :+1:

File: hr_expense_advance_clearing_sequence/data/hr_expense_data.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<odoo noupdate="1">
+
+    <record model="ir.sequence" id="seq_expense_petty_cash">
```
Comment (newtratip) [inline hr_expense_advance_clearing_sequence/data/hr_expense_data.xml]: I think id should be before model.

File: hr_expense_advance_clearing_sequence/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
```
Comment (newtratip) [inline hr_expense_advance_clearing_sequence/__init__.py]: init file not necessary for copyright

**Review by newtratip — APPROVED** [review]
LGTM :+1:

File: hr_expense_advance_clearing_sequence/data/hr_expense_data.xml (hunk 1 above)
Comment (ps-tubtim) [reply hr_expense_advance_clearing_sequence/data/hr_expense_data.xml]: Edited.

File: hr_expense_advance_clearing_sequence/__init__.py (hunk 2 above)
Comment (ps-tubtim) [reply hr_expense_advance_clearing_sequence/__init__.py]: Deleted.

**Pull request merged**